theme/layouts/theme.html.twig

```diff
@@ -35,16 +35,12 @@
             </script>
 
             <!-- Must be included before core file -->
-            <script defer src="https://cdn.jsdelivr.net/npm/@alpinejs/collapse@3.x.x/dist/cdn.min.js"></script>
-            <script defer src="https://cdn.jsdelivr.net/npm/@alpinejs/intersect@3.x.x/dist/cdn.min.js"></script>
-            <script defer src="https://cdn.jsdelivr.net/npm/@alpinejs/focus@3.x.x/dist/cdn.min.js"></script>
-            <script defer src="https://cdn.jsdelivr.net/npm/async-alpine@1.x.x/dist/async-alpine.script.js"></script>
-            <script src="https://cdn.jsdelivr.net/npm/@alpinejs/persist@3.x.x/dist/cdn.min.js"></script>
-            <script defer src="https://cdn.jsdelivr.net/npm/alpinejs@3.x.x/dist/cdn.min.js"></script>
-            <script src="https://unpkg.com/@popperjs/core@2"></script>
-
-            <!-- product spin -->
-            <script src="https://scripts.sirv.com/sirv.js" async></script>
```

Comment:
Removed sirv because this is no longer supported and removed async alpine because unused.